modules/manage/pages/manage-expiration.adoc

```diff
@@ -0,0 +1,270 @@
+The expiration value is the number of seconds in the future from the time of creation or mutation when you want the document to expire.  
+When this period of time elapses, Couchbase Server deletes the document unless you mutate the document beforehand.
+
+The expiration is part of the xref:learn:views/views-store-data.adoc#document-metadata[document's metadata]. 
+You can set the expiration when creating a document.
+This example uses SQL++ to create a new document in the `user_sessions` bucket and sets it to expire in one hour:
+
+[source, sql++]
+----
+INSERT INTO user_sessions (KEY, VALUE, OPTIONS) VALUES (
+    "123456", 
+    { 
+        "username": "jsmith@example.com",
+        "ip_address" : "192.168.10.20", 
+        "cart" : [] 
+    }, 
+    { 
+        "expiration": 3600 
+    } 
+);
+----
+
+To view the expiration of a document, query its metadata to find its `expiration` setting. 
+This SQL++ example queries the document created by the prior example using its id.
+
+[source, sql++]
+----
+SELECT meta().expiration 
+    FROM user_sessions 
+    USE KEYS '123456';
+----
+
+The result of running the previous looks like this:
+
+[source, json]
+----
+[
+  {
+    "expiration": 1704819228
+  }
+]
+----
+
+The expiration value is the UNIX-epoch timestamp when the document expires. 
+
+You can change the document's expiration by setting its expiration value during a mutation. 
+The following example updates the document with new values and also resets its expiration so it expires one hour after the statement executes.
+
+[source, sql++]
+----
+UPDATE user_sessions AS U
+    USE KEYS "123456"
+    SET meta(U).expiration = 3600, cart = [{"item": "9999", "quantity": "3"}]
+----
+
+Querying the document's metadata shows that the expiration timestamp is now later than before:
+
+[source, json]
+----
+[
+  {
+    "expiration": 1704829288
+  }
+]
+----
+
+[mutation-expiration]
+=== Mutation's Effect on Expiration
+
+By default, if you mutate a document without settings its expiration, Couchbase Server automatically sets the value to 0. 
+This value prevents the document from expiring unless the collection or bucket has a non-zero `maxTTL` setting.
+For example, suppose you execute the following SQL++ statement to update the document from the previous examples:
+
+[source, sql++]
+----
+UPDATE user_sessions
+    USE KEYS "123456"
+    SET cart = [{"item": "9999", "quantity": "3"}, {"item" : "3423", "quantity" : 1}];
+----
+
+If you then query the document's expiration metadata, you'll find that it's now 0:
+
+[source, json]
+----
+[
+  {
+    "expiration": 0
+  }
+]
+----
+
+// The following doesn't exist in 7.0.0. This should be uncommented in 7.1 and later.
+// 
+// You can prevent Couchbase Server from clearing the expiration value by using the xref:settings:query-settings.adoc#prevent_expiry[`prevent_expiry`] request-level parameter. 
+
+When mutating a document, you can set its expiration to its current expiration value. 
```

Comment:
Maybe say more plainly:
If you want to preserve a documents expiration setting when mutating it, set its expiration to its current value by...